src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java

```diff
@@ -78,6 +80,8 @@ public class GroupDialogViewModel {
     private final BooleanProperty typeTexProperty = new SimpleBooleanProperty();
 
     // Option Groups
```

Comment:
The heading does not match the following things.

Please move the Boolean Property.